src/main/java/seedu/address/model/event/Event.java

```diff
@@ -0,0 +1,137 @@
+package seedu.address.model.event;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import seedu.address.model.particpant.Participant;
+
+/**
+ * This is an Event class.
+ */
+public class Event implements Comparable<Event> {
+
+    private boolean isDone = false;
+    private ArrayList<Participant> participants = new ArrayList<>();
+    private EventName eventName;
+    private EventDate eventDate;
+    private EventTime eventTime;
+
+    /**
+     * This is the constructors of an Event.
+     *
+     * @param name of the Event.
```

Comment:
I think this parameter section is incomplete? Does checkstyle not point this out?

Reply:
For this, github does not have the syntax highlighting in the IDE, the "name" is the parameter. Apparently the code base wrote many methods like this, so I just followed 🤣